datahub-web-react/src/app/recommendations/renderer/component/CompactEntityNameList.tsx

```diff
@@ -19,6 +19,8 @@ export const CompactEntityNameList = ({ entities, onClick, linkUrlParams, showTo
     return (
         <>
             {entities.map((entity, index) => {
+                if (!entity) return <></>;
```

Comment:
same, small bug fix here ^

Reply:
Is entity not a required field? Didn't realize... Nice!